### PR TITLE
T1 30.navbar-Adjustments

### PR DESCRIPTION
**Pull request by Cortesjohnj**
# NavBar Adjustments
## Features
- The Donation button only appears when the user is logged in or no one is authenticated.
- Added Pets button, to see the list of pets only for foundations.
- Corrected the profile link.
- Footer and sidebar were adjusted with the same changes.
Nolog1
Nolog2
NologFooter
userLog1
use
![foundationLogfooter

foundationLog2
rLog
foundationLog1
footer](https://user-images.githubusercontent.com/66749007/134395237-cdca60a9-8f0a-4242-93c1-e5ca75d29f1e.jpg)
userLog2

### CODE REVIEW

File: src/assets/styles/PetForm.css [hunk 1]
```diff
@@ -0,0 +1,214 @@
+.petFormContainer {
```
Comment (leoneloliveros) [inline src/assets/styles/PetForm.css:1]: This is practically the same as petFormContainer2 so you can use one petFormContainer for both implementation and create new selector with variations (grid-template-columns: 1fr 1fr;)

File: src/assets/styles/PetForm.css [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+  box-shadow: 0 5px 8px 0 rgba(0, 0, 0, 0.2), 0 7px 20px 0 rgba(0, 0, 0, 0.2);
+  position: relative;
+  border-radius: 10px;
+  min-height: 100vh;
+  display: grid;
+}
+
+@media (max-width: 768px) {
+  .petFormContainer {
+    margin: 30px 30px;
```
Comment (leoneloliveros) [inline src/assets/styles/PetForm.css:24]: Same here!

File: src/components/pet-form/usePetForm.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  });
+  const [errors, setErrors] = useState({});
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setValues({
+      ...values,
+      [name]: value,
+      petId: petId,
```
Comment (leoneloliveros) [inline src/components/pet-form/usePetForm.js:20]: You can use only petId

File: src/assets/styles/PetForm.css (hunk 1 above)
Comment (Cortesjohnj) [reply src/assets/styles/PetForm.css:1]: Iv'e just implemented a selector only for grid-template-columns: 1fr 1fr, and used only one petContainer

File: src/components/pet-form/usePetForm.js (hunk 3 above)
Comment (Cortesjohnj) [reply src/components/pet-form/usePetForm.js]: Changed to petId only

**Review by leoneloliveros — APPROVED** [review]
Great implementation! Approved

**Pull request merged**